src/io/matpower.jl

```diff
@@ -365,6 +368,36 @@ function matpower_to_powermodels(mp_data::Dict{String,Any})
 end
 
 
+"""
+    split_loads_shunts(data)
+
+Seperates Loads and Shunts in `data` under separate "load" and "shunt" keys in the
+PowerModels data format. Includes references to originating bus via "load_bus"
+and "shunt_bus" keys, respectively.
+"""
+function split_loads_shunts(data::Dict{String,Any})
+    data["load"] = []
+    data["shunt"] = []
+
+    load_num = 1
+    shunt_num = 1
+    for (i,bus) in enumerate(data["bus"])
+        if bus["pd"] != 0.0 || bus["qd"] != 0.0
+            append!(data["load"], [Dict{String,Any}("pd" => bus["pd"], "qd" => bus["qd"], "load_bus" => bus["bus_i"], "status" => bus["bus_type"] != 4, "index" => "$load_num")])
+            load_num += 1
```

Comment:
If I recall the convention correctly, the type of `index` should be an int not a string.